aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -322,6 +322,36 @@ define([],function() {
       DOWNLOAD_NODE: "ALF_DOWNLOAD_FILE",
 
       /**
+       * This is the response topic for requests to obtain node data that will be immediately
+       * followed by the downloading of that node. It is set as the response topic for smart
+       * downloads of nodes included in Search API requests where the "nodeRef" attribute
+       * is available but the "contentURL" attribute is not. This is not intended to be used by 
+       * widgets or services other than the 
+       * [DocumentService]{@link module:alfresco/services/DocumentService}.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.43
+       */
```

Comment:
I'm not sure this is an event... since it is neither subscribed to, nor published... it is simply a response scope. The actual subscription and publication is the DOWNLOAD_ON_NODE_RETRIEVAL_SUCCESS which is why I didn't give it the @event tag.